[ INFO] [1720863322.699820161] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.699849933] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 0
[ INFO] [1720863322.699863796] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.699903405] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 8, spaceOccupied: 9.000000
[ERROR] [1720863322.699923790] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 8
[ERROR] [1720863322.699942238] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.700075205] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.700090142] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 0
[ INFO] [1720863322.700102216] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.700116510] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 7, spaceOccupied: 9.000000
[ERROR] [1720863322.700203348] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 7
[ERROR] [1720863322.700217152] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.700232500] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 7, spaceOccupied: 9.000000
[ERROR] [1720863322.700245964] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 7
[ERROR] [1720863322.700258868] [/multi_robot_router] [675]: occupation.robot == _robot
[ERROR] [1720863322.700273287] [/multi_robot_router] [679]: occupation.startTime >= _potential
[ INFO] [1720863322.700287009] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 7, spaceOccupied: 18.000000
[ERROR] [1720863322.700300539] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 7
[ERROR] [1720863322.700313916] [/multi_robot_router] [679]: occupation.startTime >= _potential
[ INFO] [1720863322.700327641] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 7, spaceOccupied: 18.000000
[ERROR] [1720863322.700340804] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 7
[ERROR] [1720863322.700354098] [/multi_robot_router] [679]: occupation.startTime >= _potential
[ INFO] [1720863322.700397666] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.700411651] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 0
[ INFO] [1720863322.700424954] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.700439479] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 32, spaceOccupied: 9.000000
[ERROR] [1720863322.700453540] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.700466643] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.700479891] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 32, spaceOccupied: 9.000000
[ERROR] [1720863322.700493526] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.700506433] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.700520070] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 32, spaceOccupied: 9.000000
[ERROR] [1720863322.700532608] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.700546252] [/multi_robot_router] [675]: occupation.robot == _robot
[ERROR] [1720863322.700561066] [/multi_robot_router] [679]: occupation.startTime >= _potential
[ INFO] [1720863322.700575215] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 32, spaceOccupied: 18.000000
[ERROR] [1720863322.700620352] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ INFO] [1720863322.700635268] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 32, spaceOccupied: 18.000000
[ERROR] [1720863322.700650919] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.700664420] [/multi_robot_router] [679]: occupation.startTime >= _potential
[ INFO] [1720863322.700679782] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 32, spaceOccupied: 18.000000
[ERROR] [1720863322.700693819] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.700706802] [/multi_robot_router] [679]: occupation.startTime >= _potential
{1, 93}
[ WARN] [1720863322.700784227] [/multi_robot_router] [109]: ROBOT : 1, SEGMENTS ID: 32, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 3493798491261196207906441833170020823069260415342381089300571330259573897977757596980526675064474858628474642169408852698516123056011333303855979387221269074667508180503178700318739217163650309754071585863446293247340277059813376.000000
[ WARN] [1720863322.700981218] [/multi_robot_router] [109]: ROBOT : 1, SEGMENTS ID: 32, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.700998925] [/multi_robot_router] [109]: ROBOT : 1, SEGMENTS ID: 32, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ERROR] [1720863322.701050101] [/multi_robot_router] [116]: updatePreconditions robotId: 1,  stepCondition: 2
[ INFO] [1720863322.701078001] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.701100299] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 0
[ INFO] [1720863322.701112983] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.701126466] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 33, spaceOccupied: 9.000000
[ERROR] [1720863322.701141274] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 33
[ERROR] [1720863322.701154839] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.701172226] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 33, spaceOccupied: 9.000000
[ERROR] [1720863322.701187356] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 33
[ERROR] [1720863322.701206649] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.701218458] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 33, spaceOccupied: 18.000000
[ERROR] [1720863322.701238320] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 33
[ INFO] [1720863322.701255348] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 33, spaceOccupied: 18.000000
[ERROR] [1720863322.701270992] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 33
[ERROR] [1720863322.701283987] [/multi_robot_router] [679]: occupation.startTime >= _potential
{1, 124}
[ WARN] [1720863322.701308108] [/multi_robot_router] [109]: ROBOT : 1, SEGMENTS ID: 33, POTENTIAL PATH: 124.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.701324928] [/multi_robot_router] [109]: ROBOT : 1, SEGMENTS ID: 33, POTENTIAL PATH: 124.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.701340340] [/multi_robot_router] [109]: ROBOT : 1, SEGMENTS ID: 33, POTENTIAL PATH: 124.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.701356165] [/multi_robot_router] [109]: ROBOT : 1, SEGMENTS ID: 33, POTENTIAL PATH: 124.000000, POTENTIAL LINETIME: 0.000000
[ERROR] [1720863322.701369431] [/multi_robot_router] [116]: updatePreconditions robotId: 1,  stepCondition: 3
[ INFO] [1720863322.701478739] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.701494500] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 1
[ INFO] [1720863322.701506781] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.701522507] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 6, spaceOccupied: 18.000000
[ERROR] [1720863322.701537071] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 6
[ERROR] [1720863322.701548584] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.701567507] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.701581709] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 1
[ INFO] [1720863322.701592712] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.701606106] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 5, spaceOccupied: 18.000000
[ERROR] [1720863322.701619894] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 5
[ERROR] [1720863322.701632283] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.701649266] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.701665073] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 1
[ INFO] [1720863322.701678146] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.701695976] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 4, spaceOccupied: 9.000000
[ERROR] [1720863322.701715612] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 4
[ INFO] [1720863322.701732180] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 4, spaceOccupied: 9.000000
[ERROR] [1720863322.701751844] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 4
[ INFO] [1720863322.701767513] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 4, spaceOccupied: 18.000000
[ERROR] [1720863322.701785060] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 4
[ERROR] [1720863322.701804622] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.701823779] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 4, spaceOccupied: 18.000000
[ERROR] [1720863322.701839925] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 4
[ERROR] [1720863322.701858325] [/multi_robot_router] [675]: occupation.robot == _robot
[ERROR] [1720863322.701873706] [/multi_robot_router] [679]: occupation.startTime >= _potential
{0, 62}
{0, 93}
[ WARN] [1720863322.701898835] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 4, POTENTIAL PATH: 62.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.701917696] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 4, POTENTIAL PATH: 62.000000, POTENTIAL LINETIME: 0.000000
[ERROR] [1720863322.701938246] [/multi_robot_router] [116]: updatePreconditions robotId: 0,  stepCondition: 1
[ WARN] [1720863322.701975747] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 4, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 7190544425112409442868925081015051557360941132157715771871668262070989252119119259516618417654755315124084199898987290655457651868977604911838728040740383301952652679455110745457079335550093750100757442769738174393613760552933629550409137279993116622848.000000
[ WARN] [1720863322.701998173] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 4, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.702015723] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 4, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ERROR] [1720863322.702049383] [/multi_robot_router] [116]: updatePreconditions robotId: 0,  stepCondition: 2
[ INFO] [1720863322.702074205] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.702087795] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 1
[ INFO] [1720863322.702101760] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.702116311] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 32, spaceOccupied: 9.000000
[ERROR] [1720863322.702133101] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ INFO] [1720863322.702151278] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 32, spaceOccupied: 9.000000
[ERROR] [1720863322.702168862] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ INFO] [1720863322.702185495] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 32, spaceOccupied: 9.000000
[ERROR] [1720863322.702201375] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ INFO] [1720863322.702217773] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 32, spaceOccupied: 18.000000
[ERROR] [1720863322.702234359] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.702247921] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.702264349] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 32, spaceOccupied: 18.000000
[ERROR] [1720863322.702280221] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.702296398] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.702315474] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 32, spaceOccupied: 18.000000
[ERROR] [1720863322.702332836] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 32
[ERROR] [1720863322.702347964] [/multi_robot_router] [675]: occupation.robot == _robot
[ERROR] [1720863322.702362986] [/multi_robot_router] [679]: occupation.startTime >= _potential
{0, 62}
{0, 93}
{0, -1}
[ WARN] [1720863322.702396243] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 32, POTENTIAL PATH: 62.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.702411752] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 32, POTENTIAL PATH: 62.000000, POTENTIAL LINETIME: 0.000000
[ERROR] [1720863322.702428881] [/multi_robot_router] [116]: updatePreconditions robotId: 0,  stepCondition: 1
[ WARN] [1720863322.702471582] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 32, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 7190544425112409442868925081015051557360941132157715771871668262070989252119119259516618417654755315124084199898987290655457651868977604911838728040740383301952652679455110745457079335550093750100757442769738174393613760552933629550409137279993116622848.000000
[ WARN] [1720863322.702489756] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 32, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.702504712] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 32, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ERROR] [1720863322.702527082] [/multi_robot_router] [116]: updatePreconditions robotId: 0,  stepCondition: 2
[ WARN] [1720863322.702563391] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 32, POTENTIAL PATH: -1.000000, POTENTIAL LINETIME: 7190544425112409442868925081015051557360941132157715771871668262070989252119119259516618417654755315124084199898987290655457651868977604911838728040740383301952652679455110745457079335550093750100757442769738174393613760552933629550409137279993116622848.000000
[ERROR] [1720863322.702579052] [/multi_robot_router] [116]: updatePreconditions robotId: 0,  stepCondition: 0
[ INFO] [1720863322.702597843] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.702618507] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 1
[ INFO] [1720863322.702634375] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.702649082] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 35, spaceOccupied: 9.000000
[ERROR] [1720863322.702665394] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 35
[ INFO] [1720863322.702680652] [/multi_robot_router] [654]: ROBOT ID: 0 in timeline segId: 35, spaceOccupied: 9.000000
[ERROR] [1720863322.702697088] [/multi_robot_router] [658]: add getListOfRobotsHigherPrioritizedRobots _segId : 35
[ INFO] [1720863322.702713857] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 35, spaceOccupied: 18.000000
[ERROR] [1720863322.702730321] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 35
[ERROR] [1720863322.702743641] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.702757458] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 35, spaceOccupied: 18.000000
[ERROR] [1720863322.702773469] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 35
[ERROR] [1720863322.702786504] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.702803180] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 35, spaceOccupied: 18.000000
[ERROR] [1720863322.702818213] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 35
[ERROR] [1720863322.702833327] [/multi_robot_router] [675]: occupation.robot == _robot
[ERROR] [1720863322.702846654] [/multi_robot_router] [679]: occupation.startTime >= _potential
{0, 93}
{0, -1}
[ WARN] [1720863322.702879632] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 35, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.702895271] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 35, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ WARN] [1720863322.702910630] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 35, POTENTIAL PATH: 93.000000, POTENTIAL LINETIME: 0.000000
[ERROR] [1720863322.702926192] [/multi_robot_router] [116]: updatePreconditions robotId: 0,  stepCondition: 2
[ WARN] [1720863322.703238383] [/multi_robot_router] [109]: ROBOT : 0, SEGMENTS ID: 35, POTENTIAL PATH: -1.000000, POTENTIAL LINETIME: 7190544425112409442868925081015051557360941132157715771871668262070989252119119259516618417654755315124084199898987290655457651868977604911838728040740383301952652679455110745457079335550093750100757442769738174393613760552933629550409137279993116622848.000000
[ERROR] [1720863322.703292937] [/multi_robot_router] [116]: updatePreconditions robotId: 0,  stepCondition: 0
[ INFO] [1720863322.703321832] [/multi_robot_router] [51]: -----------------------------------------------
[ INFO] [1720863322.703336561] [/multi_robot_router] [52]: UPDATE PRECONDITIONS ROBOT : 1
[ INFO] [1720863322.703351665] [/multi_robot_router] [53]: -----------------------------------------------
[ INFO] [1720863322.703365111] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 42, spaceOccupied: 18.000000
[ERROR] [1720863322.703376354] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 42
[ERROR] [1720863322.703388090] [/multi_robot_router] [675]: occupation.robot == _robot
[ INFO] [1720863322.703400355] [/multi_robot_router] [654]: ROBOT ID: 1 in timeline segId: 42, spaceOccupied: 18.000000
[ERROR] [1720863322.703412216] [/multi_robot_router] [672]: Dont need to add getListOfRobotsHigherPrioritizedRobots _segId : 42
[ERROR] [1720863322.703423094] [/multi_robot_router] [675]: occupation.robot == _robot
[ WARN] [1720863322.703498596] [/multi_robot_router] [154]: usePriorityRescheduler_
[ WARN] [1720863322.703513865] [/multi_robot_router] [158]: duration < _timeLimit
[ WARN] [1720863322.703527477] [/multi_robot_router] [162]: found
[ INFO] [1720863322.703692029] [/multi_robot_router] [452]: /multi_robot_router: Plan found :-), publishing plan
[ INFO] [1720863322.703903168] [/multi_robot_router] [399]:
Success 1, 1 = 1.000, avr   58 ms, success:   58 ms, PR= on, SR= on, CR= on
 [1.000,   58,    58]
[ WARN] [1720863322.900621410] [/robot_1/local_behavior] [184]: init when receive new route robot name = robot_1, current_route_segment = -1
[ WARN] [1720863322.900754612] [/robot_1/local_behavior] [222]: WAIT FOR RESET current_route_segment
[ WARN] [1720863322.964885137] [/robot_0/local_behavior] [184]: init when receive new route robot name = robot_0, current_route_segment = -1
[ INFO] [1720863324.596797594] [/robot_0/trajectory_controller] [45]: Multi Robot Controller: goal reached
[ INFO] [1720863326.996709114] [/robot_1/trajectory_controller] [45]: Multi Robot Controller: goal reached